code/game/machinery/doors/blast_door.dm

```diff
@@ -147,6 +150,10 @@
 	if (src.operating || (stat & BROKEN || stat & NOPOWER))
 		return
 	force_close()
+	for(var/turf/turf in locs)
+		for(var/atom/movable/AM in turf)
+			if(AM.airlock_crush(damage))
+				take_damage(damage)
```

Comment:
This proc causes the door to take damage. A standard door has 300 health. This is, what, 8 ~~crushings~~ objects crushed before the door poops itself? I'd kinda expect them to be sturdier.